chemprop/uncertainty/predictor.py

```diff
@@ -9,7 +9,11 @@
 from chemprop.utils.registry import ClassRegistry
 
 
-class UncertaintyPredictor:
+class UncertaintyPredictor(ABC):
+    """
+    A class for making model predictions and associated uncertainty predictions.
+    """
+
     def __call__(self, dataloader: DataLoader, models: Iterable[MPNN], trainer: pl.Trainer):
         return self._calc_prediction_uncertainty(dataloader, models, trainer)
```

Comment:
if this method is just syntactic sugar for `_calc_prediction_uncertainty`, why do we have the latter method? Can't we just rename it?

Reply:
The intention is to match the style more closely with the metrics module, but there are no new operations in `__call__` other than `_calc_prediction_uncertainty` in the current implementation. I will rename it to `__call__`.